web/templates/gear_menu_popover.hbs

```diff
@@ -0,0 +1,189 @@
+<div class="navbar-dropdown-menu" id="gear-menu-dropdown" aria-labelledby="settings-dropdown" data-simplebar>
+    <ul class="navbar-dropdown-menu-outer-list">
+        <li class="org-info">
```

Comment:
I LOVE!!! to see this nice nested list semantics.

I think it would be useful to have identifying class names on each of these top-level `<li>` elements and possibly a shared one, too, across all of them. I've got come CSS comments to write in a moment on that topic.

Reply:
I added shared class names for ul / li tags. Having a specific class name for each felt adding a lot of unused class names.

Reply:
Done.